scripts/start.js

```diff
@@ -41,6 +41,10 @@ const suggest = (input, choices) => Promise.resolve(choices.filter(i => i.title.
     {
       shell: true,
       stdio: 'inherit',
+      env: {
+        ...process.env,
+        NODE_OPTIONS: '--max-old-space-size=4096',
```

Comment:
**q:** why do we need implicitly squeeze every devs machine RAM ?

Reply:
Because Node's default low limit is causing crashes in some cases, and this is the only immediate workaround. (Investigating how to decrease the memory usage is worthwhile long-term, but not a feasible short-term workaround.) Or did you mean limit use of the workaround to the cases where it's specifically known to be needed?

Reply:
I removed the setting except for in `@fluentui/react` (where it's definitely needed for the moment)